lib/project_types/script/layers/infrastructure/languages/assemblyscript_task_runner.rb

```diff
@@ -47,6 +47,12 @@ def metadata
             Domain::Metadata.create_from_json(@ctx, raw_contents)
           end
 
+          def library_version(library_name)
+            output, status = ctx.capture2e("npm list --json")
```

Comment:
should we use our [command runner](https://github.com/Shopify/shopify-cli/blob/main/lib/project_types/script/layers/infrastructure/command_runner.rb) here?